app/templates/items/index.hbs

```diff
@@ -23,26 +23,25 @@
   <div class="row search">
     <div class="small-12 columns">
       {{#if displayResults}}
+
         {{#infinite-list height="75vh" loadMore=(action "loadMoreItems") as |items| }}
           <ul class="list list-activity list-offer-items">
             {{#each items as |item|}}
               {{partial 'orders/item_block'}}
             {{/each}}
```

Comment:
@shreyas098 , you can use `{{else}}` inside each block. 

see [this](https://guides.emberjs.com/release/components/looping-through-lists/#toc_empty-lists-1)

Reply:
Done